frontend/src/metabase/common/components/AggregationPicker/AggregationPicker.tsx

```diff
@@ -429,21 +433,12 @@ function getCompareListItem(
     return undefined;
   }
 
-  const period = getOffsetPeriod(query, stageIndex);
-
-  if (aggregations.length > 1) {
-    return {
-      type: "compare",
-      displayName: t`Compare to previous ${period} ...`,
-    };
-  }
-
-  const [aggregation] = aggregations;
-  const info = Lib.displayInfo(query, stageIndex, aggregation);
-
   return {
-    type: "compare",
-    displayName: t`Compare “${info.displayName}” to previous ${period} ...`,
+    type: "action",
+    key: "compare",
+    name: t`Compare to the past`,
```

Comment:
This might not be the right terminology when the user wants to compare non-time columns.

@mngr do you think that usecase is important enough or should we change the wording here?